config/confighttp/confighttp.go

```diff
@@ -479,13 +481,14 @@ func (hss *ServerConfig) ToServer(_ context.Context, host component.Host, settin
 		next:            handler,
 		includeMetadata: hss.IncludeMetadata,
 	}
-
+	errorLog, _ := zap.NewStdLogAt(settings.Logger, zapcore.ErrorLevel)
```

Comment:
can we return the error?

Reply:
I dont think so because if errLog is nil in http.Server it will default to standard logger but if we return the error it may disrupt the entire flow In jaeger  also similar pattern is followed
https://github.com/jaegertracing/jaeger/blob/8f612a876dc501e0db9786a054ff52fee314afb3/cmd/collector/app/server/http.go#L47

Reply:
it's fine to return the error, but the error handling won't be tested because the error will never happen - it only happens if the 2nd argument is not a valid log level, e.g. `zapcore.Level(123)`

Reply:
sure then will make a change returning the error

Reply:
it only matters if zap changes its internals moving forward, and since we're returning an error anyway, might as well return it.

Reply:
Hey Just a small question Unrelated to this 
I want to contribute to the OpenTelemetry ecosystem actively. Which repositories would be most suitable for someone at a beginner level to start contributing effectively?